sprocs/workspace_loads_current.sql

```diff
@@ -0,0 +1,120 @@
+CREATE OR REPLACE FUNCTION
+    workspace_loads_current(
+        IN workspace_capacity_factor double precision,
+        IN workspace_host_capacity integer,
```

Comment:
I suggest making this a double -- not really any reason not to and over time I've found it's sometimes painful to lock ourselves in to integers.